solr/bin/solr.cmd

```diff
@@ -246,11 +246,12 @@ set FIRST_ARG=%1
 
 IF [%1]==[] goto usage
 
-IF "%1"=="-help" goto usage
-IF "%1"=="-usage" goto usage
-IF "%1"=="-h" goto usage
-IF "%1"=="--help" goto usage
-IF "%1"=="/?" goto usage
+IF "%1"=="-help" goto run_solrcli
+IF "%1"=="-usage" goto run_solrcli
+IF "%1"=="-h" goto run_solrcli
+IF "%1"=="--help" goto run_solrcli
+IF "%1"=="-help" goto run_solrcli
+IF "%1"=="/?" goto run_solrcli
 IF "%1"=="status" goto get_status
 IF "%1"=="version" goto run_solrcli
 IF "%1"=="-v" goto run_solrcli
```

Comment:
Regarding Windows script, we either need a volunteer who can use som cmd-fu to bring it up to speed.

Or, perhaps we can prune it from 2100 lines to a fraction by using it as a test bench for delegation to `SolrCLI`.
* Remove most arg parsing and just pass raw args to `run_solrcli`.
* Remove all or most usage help and let SolrCLI handle it
* Remove most ENV translation to sysprop since `EnvUtils` will handle it.

Doing so will not be 100% back compat wrt what the script prints, but it would be a test bed for improving the Java side of help formatting and arg parsing, ensuring 100% equal CLI API between Linux and Windows.

I'd rather slim down solr.cmd to something we can manage and risk some back-compat issues than ship a defunct script we cannot maintain.

Reply:
I think your proposal makes sense...   how can we capture this as a decision point?  Maybe open a JIRA that specifies this work to be done?   Basically, how do we make sure the idea doesn't get dropped if no one does it immediately and I am not back for three weeks ;-)

Reply:
The other PR with better help formatting is an enabler..

Maybe add a handful new sub-tasks to your umbrella JIRA - not sure if one JIRA per area (migrate usage) or one JIRA per tool (healthcheck etc) makes the most sense. Maybe one per tool, and we do one PR as an example and invite dev-list to pick the others? 

Removing usesless ENV translation can have one JIRA and multiple PRs.

Reply:
There is already a JIRA SOLR-15831 umbrella for much of this